### ISSUE TITLE
error

### ISSUE THREAD

**Issue by Jvr2022** [YABOIpy/GoDm#32]
2024/01/16 12:29:30 Post "https://discord.com/api/v9/channels/1195080902112448666/messages": unexpected EOF
Going Back To Menu =>..

**Comment by YABOIpy** [comment]
Context..

**Comment by Jvr2022** [comment]
That was the error

**Comment by Jvr2022** [comment]
2024/01/16 12:29:30 Post "https://discord.com/api/v9/channels/1195080902112448666/messages": unexpected EOF
Going Back To Menu =>..

**Comment by YABOIpy** [comment]
Yeah cause im supposed to sniff exactly what ur doing.. read the readme before making an issue
